provider/glencoe_check.py

```
@@ -60,6 +62,30 @@ def metadata(msid, settings):
     
     return resp.json()
 
+
+def jpg_href_values(metadata):
```

as in standup, we should fail fast if the filenames provided by Glencoe are not what we expect. This check is not blocking for this PR, but should be in the DOD